### PR TITLE
Add compat for <angle> CSS data type / angle units

### PR DESCRIPTION
**Pull request by Elchi3**
This sets up CSS data types / units. I think this structure makes sense:
- css 
  - types
    - angle
      - deg
      - rad
    - length
      - px
      - em
    - frequence
      - hz
      - khz

This PR only adds \<angle>. Please review if the structure makes sense, so we can go forward with more CSS data types and/or units.

### CODE REVIEW

File: css/types/angle.json [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": "2"
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
```
Comment (teoli2003) [inline css/types/angle.json]: "12"

File: css/types/angle.json [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "3.6"
```
Comment (teoli2003) [inline css/types/angle.json]: Sure about 3.6? transform has been added in 3.5 and angles are used for some functions like skew()

File: css/types/angle.json [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+              "version_added": true
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": "4"
```
Comment (teoli2003) [inline css/types/angle.json]: Sure about 4? transform has been added in 3.1 and angles are used for some functions like skew()

File: css/types/angle.json [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+            },
+            "safari_ios": {
+              "version_added": null
```
Comment (teoli2003) [inline css/types/angle.json]: This is definitively a true.

File: css/types/angle.json [hunk 5]
```diff
@@ -0,0 +1,262 @@
+{
+  "css": {
+    "types": {
+      "angle": {
```
Comment (teoli2003) [inline css/types/angle.json]: The comments at the type level also apply to the units (except turn)

File: css/types/angle.json (hunk 2 above)
Comment (Elchi3) [reply css/types/angle.json]: This is what is written in the current table. I'm no expert and I haven't invested time to do research.

File: css/types/angle.json (hunk 1 above)
Comment (Elchi3) [reply css/types/angle.json]: Changed.

File: css/types/angle.json (hunk 3 above)
Comment (Elchi3) [reply css/types/angle.json]: This is what is written in the current table. I'm no expert and I haven't invested time to do research.

File: css/types/angle.json (hunk 4 above)
Comment (Elchi3) [reply css/types/angle.json]: Changed.

File: css/types/angle.json (hunk 2 above)
Comment (teoli2003) [reply css/types/angle.json]: Ok, our goal is to migrate, not necessary to make the content perfect. We will have special activities for this in 2018.

File: css/types/angle.json (hunk 3 above)
Comment (teoli2003) [reply css/types/angle.json]: Ok, our goal is to migrate, not necessary to make the content perfect. We will have special activities for this in 2018.

**Pull request merged**